warehouse/filters.py

```diff
@@ -136,3 +137,8 @@ def format_classifiers(classifiers):
     structured = collections.OrderedDict(sorted(structured.items()))
 
     return structured
+
+
+def filter_valid_uris(items):
+    """ Filter out a list so that it only contains valid urls """
```

Comment:
Hadn't seen that, thanks! (Now I have to update https://github.com/controversial/livejson/ so that I can still brag about PEP compliance...)